src/modules/filters/services.js

```diff
@@ -1,8 +1,19 @@
-import httpClient from '../../services/http-client'
+import request from '../../services/request-backend'
+import storageService from '../../services/storage-service'
 
+const enforceArray = element => (Array.isArray(element) ? element : [])
 
 export default {
-  list() {
-    return httpClient.get('filters/')
-  },
+  list: () =>
+    request.get('filters', {
+      headers: {
+        Authorization: `Token ${storageService().get('auth-token')}`,
```

Comment:
Please don't do this. I spent literally a few days to refactor a similar code on another project.
If you MUST use localstorage to persist the token, get it when bootstraping the app and save it to the store. Than you always get it from the store on the `action`, not the service.

BTW, we should have a second param named `key` that sets the header for you.